README.md

```diff
@@ -26,9 +26,9 @@ but it's probably not a Ruby conference.
 
 The list of events is driven by the conferences file in the `_data` directory - if you have an update for those things, just change the YAML and send a PR.
 
-The file to be changed is `_data/conferences.yml`. **NOTE: It is
-order-dependent, meaning, put your conference in the YAML file in the order in
-which it should appear on the page, which is chronological.**
+The file to be changed is `_data/conferences.yml`. It is NOT order-dependent.
+Put your conference in the YAML file at the end.
+The page will sort the conferences by `start_date`.
```

Comment:
I was wondering if we should still encourage people to actually add the conferences in the right spot, even if it's going to apply the sort anyway.

Reply:
@marcoroth interesting. What would be the goal with that kind of documentation? I was thinking this would alleviate the need for this requirement, but I'm curious to hear more feedback.

Reply:
I was more thinking that it continues to be consistent in the file itself, like if you are not looking at the data through the website. Up to this point it's been in the right order, so it would be weird if anything after now wouldn't.

But anyway, this isn't a show-stopper as it yields the same end-result.